spec/controllers/beta/projects_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe Beta::ProjectsController do
+  context 'when logged out' do
+    specify do
+      get :show, params: { id: 42 }
+      expect(response).to redirect_to(new_user_session_url)
+    end
+  end
+
+  describe 'when normal user is logged in' do
+    let(:normal_user)    { create :user, :with_team, email: 'normal_user@example.com' }
+    let(:normal_team)    { normal_user.teams.first }
+    let(:normal_project) { create :project, users: [normal_user] }
+
+    before do
+      normal_team.ownerships.create(project: normal_project, is_owner: true)
+      sign_in normal_user
+    end
+
+    context 'when normal user try to access beta version' do
```

Comment:
how about change to `when a normal user tries to access the beta version` so it reads a little easier?